
{'date': 'Sat Oct 26 2019 12:06:16.246 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 4
  elemento =
             ^
SyntaxError: invalid syntax
'''},
{'date': 'Sat Oct 26 2019 12:10:32.342 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 8
  cenaholmes.vai()
  ^
IndentationError: unexpected indent
'''},
{'date': 'Sat Oct 26 2019 12:11:20.698 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 8
  cenaholmes.vai()
  ^
IndentationError: unexpected indent
'''},
{'date': 'Sat Oct 26 2019 12:12:06.929 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 8
  cenaholmes.vai()
  ^
IndentationError: unexpected indent
'''},
{'date': 'Sat Oct 26 2019 12:13:26.98 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 8
  cenaholmes.vai()
  ^
IndentationError: unexpected indent
'''},
{'date': 'Sat Oct 26 2019 12:14:47.454 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 2
    from _spy.vitollino.main import cena, Elemento, Texto
ImportError: cannot import name 'cena'

ImportError
Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 264
    action()
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 2
    from _spy.vitollino.main import cena, Elemento, Texto
'''},
{'date': 'Sat Oct 26 2019 12:15:24.7 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 2
    from _spy.vitollino.main import cena, Elemento, Texto
ImportError: cannot import name 'cena'

ImportError
Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 264
    action()
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 2
    from _spy.vitollino.main import cena, Elemento, Texto
'''},
{'date': 'Sat Nov 09 2019 11:08:41.557 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 14
  elem.vai = txtElem.vai
  ^
IndentationError: unexpected indent
'''},
{'date': 'Sat Nov 09 2019 11:10:20.271 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 14
  elem.vai = txtElem.vai
  ^
IndentationError: unexpected indent
'''},
{'date': 'Sat Nov 09 2019 11:12:09.693 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 14
  elem.vai = txtElem.vai
  ^
IndentationError: unexpected indent
'''},